CRM/Contact/BAO/Group.php

```diff
@@ -1425,7 +1237,7 @@ public static function getChildGroupIds($regularGroupIDs) {
    * @return int
    */
   public static function filterActiveGroups($parentArray) {
-    if (count($parentArray) > 1) {
+    if (count($parentArray) >= 1) {
```

Comment:
@colemanw there is a test failure which seems to have exposed a internal issue and encountered now because of moving the code into pre/post hook esp the code that handles parent groups. The test failure is at `CRM_Contact_BAO_GroupTest::testGroupHirearchy` - test if both parent A and B of child C are disabled then fetching group hierarchy for group C shouldn't (i.e. `CRM_Contact_BAO_Group::getGroupsHierarchy(['group-id C' => 1])`) have group C in it. This change is to check if the child group has atleast one active parent group instead of considering child group that has more then two parent groups. The test failure was at this [point](https://github.com/civicrm/civicrm-core/blob/master/tests/phpunit/CRM/Contact/BAO/GroupTest.php#L104)